org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/MonitorTest.java

```diff
@@ -340,7 +343,8 @@ private void assertPerScanMonitors(IRunnableDevice<ScanModel> scanner, List<Stri
 		final ScanModel  scanModel = new ScanModel();
 		scanModel.setPositionIterable(gen);
 		scanModel.setDetectors(detector);
-		scanModel.setMonitors(createMonitors(monitorNames));
+		scanModel.setMonitorsPerPoint(createMonitors(monitorNamesPerPoint));
+		scanModel.setMonitorsPerScan(createMonitors(monitorNamesPerScan));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)